alchemist-loading/src/main/kotlin/it/unibo/alchemist/loader/export/extractors/ExecutionTime.kt

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (C) 2010-2021, Danilo Pianini and contributors
+ * listed in the main project's alchemist/build.gradle.kts file.
+ *
+ * This file is part of Alchemist, and is distributed under the terms of the
+ * GNU General Public License, with a linking exception,
+ * as described in the file LICENSE in the Alchemist distribution's top directory.
+ */
+
+package it.unibo.alchemist.loader.export.extractors
+
+import it.unibo.alchemist.loader.export.Extractor
+import it.unibo.alchemist.model.interfaces.Environment
+import it.unibo.alchemist.model.interfaces.Reaction
+import it.unibo.alchemist.model.interfaces.Time
+
+/**
+ * An extractor which provides informations about the running time of the simulation.
+ *
+ */
+class ExecutionTime : Extractor<Double> {
```

Comment:
Wait, what is the scope of this?
I hope it's not meant to be logged when `time` is required, is it?

Reply:
No, it's a custom export and it logs the running time in seconds.
In order to log it you have to specify in the export section the type as `ExecutionTime`.